Zype/Const.swift

```diff
@@ -50,6 +50,9 @@ class Const: NSObject {
     static let kLiveItemEnabled = false
     static let kLiveVideoID = "5c8faa021d1f4314dd006203"
     static let kEpisodeNumberDisplay = false
+    static let kSegmentAnalytics = false // if enabled, make sure kSegmentAccountID has a value entered
+    static let kSegmentAnalyticsWriteKey = "enter_write_key_here" // must have some value if kSegmentAnalytics is enabled
```

Comment:
This value should not be hardcoded. It needs to be entered by the customer. I've removed it from the commit.